tests/testapp/test_helpers.py

```diff
@@ -15,35 +15,37 @@ def __new__(cls, id_):
         return super(WithId, cls).__new__(cls, id_, 'type name')
 
 
-def run_testcase_with_django_runner(
```

Comment:
@zsoldosp please see here for the latest changes, I fixed both points: https://github.com/PaesslerAG/django-performance-testing/pull/13/commits/694ec5a0ccd0bee39305f7f188e5b6ddf43773c5